modules/websearch/lib/search_engine.py

```diff
@@ -57,6 +57,7 @@
      CFG_SCOAP3_SITE, \
      CFG_OAI_ID_FIELD, \
      CFG_WEBCOMMENT_ALLOW_REVIEWS, \
+     CFG_WEBSEARCH_ADVANCED_SEARCH_EXACT_MATCH, \
```

Comment:
Please make a try/except when importing this new configuration variable, otherwise we might break some existing 1.2.x instances.